data/blog/remote-control-vscode-release-recap-sept-2023.mdx

```diff
@@ -0,0 +1,55 @@
+
+For more, check out our video demo:
+
+<figure>
+  <iframe
+    width="560"
+    height="315"
+    src="https://www.youtube.com/embed/ASZm_HyrHIc?si=bStK4K3J-Qclj59E"
+    title="YouTube video player"
+    frameBorder="0"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+    allowfullscreen
+  ></iframe>
+</figure>
+
+### Funnel and Serve web flows to get you started
+
+**Parker**: It’s no secret that we here love [Funnel](https://tailscale.com/kb/1223/tailscale-funnel/) (the Tailscale tool that lets you share a port on your localhost with the whole internet) and [Serve](https://tailscale.com/kb/1242/tailscale-serve/) (the super-simple server that powers Funnel and also lets you share privately with just your tailnet). But not everyone has had a chance to get acquainted with these features, and the first time activating them may require some [adjustments to your settings or your ACLs](https://tailscale.com/kb/1223/tailscale-funnel/#tailnet-policy-file-requirement).
+
+For folks who try to use Serve or Funnel without all the right configuration in place, we’ve added [a web-based guide](https://tailscale.com/kb/1223/tailscale-funnel/#setup) to take you through these steps right when you need it. True story: I hit this flow organically the other day when trying to use Serve from a tagged device for the first time — and it solved my problem in seconds. So whether you’re a first-timer or literally a Tailscale employee, we’re hoping this new system will help you spend less time reading error messages and more time sharing with your tailnet or the world.
+
+### Tailnet lock now in beta
+
+**Parker**: Tailscale is always thinking about your [security and privacy](https://tailscale.com/security/), and you can see it in the way we’ve designed our software. We use end-to-end WireGuard encryption for every tailnet connection, so we can never see your data. And we’ve given administrators a [device approval](https://tailscale.com/kb/1099/device-approval/) option, to review any new node that tries to log in to the network, which provides a level of security and convenience that works for most users.
+
+But for the most sensitive use cases, even that may not be enough. For these users, we provide tailnet lock—[now in beta](https://tailscale.com/blog/tailnet-lock-beta/)!—which requires that new node keys be verified by existing trusted nodes before other devices trust them. To be clear, this is a level of security above device approval, and even we don’t have access to those signing keys, so you have to handle how they’re stored and managed.
+
+Coming out of alpha, we’ve added some new features on that front. The beta now offers web configuration options, the ability to use an iOS device as a signing key, and a mechanism to recover tailnet control in case of a key compromise. For infosec-oriented users who are professionally paranoid—and we salute you—this is a feature you’ll want to check out.
```

Comment:
```suggestion
Coming out of alpha, we’ve added some new features on that front. The beta now offers web configuration options, the ability to use an iOS device as a signing key, and a mechanism to recover tailnet control in case of a key compromise. For infosec-oriented users who are professionally paranoid—we get it, really, our industry as a whole does a bad job at making this paranoia unfounded—this is a feature you’ll want to check out.
```